.travis.yml

```diff
@@ -21,6 +21,7 @@ install:
   ./web
 - docker build -f ./gradle/Dockerfile -t "quay.io/azavea/driver-gradle:${TRAVIS_COMMIT:0:7}"
   ./gradle
+- docker build -f ./analysis_tasks/Dockerfile -t "quay.io/azavea/driver-analysis:${TRAVIS_COMMIT:0:7}"
```

Comment:
I think that a path is missing on the end. The others seem to have the path on a new line.